minpy/dispatch/policy.py

```diff
@@ -61,15 +61,38 @@ def _decide(self, candidates, args, kwargs):
         """
         raise NotImplementedError()
 
-    def op_stat(self):
+    def _update_mxnet_stat(self, name):
```

Comment:
Use `collections.defaultdict` instead. Then no need to have two separate methods. Just a single `self._mxnet_op_stat[name] += 1`.